src/SeismicVelocity/SeismicVelocity.jl

```diff
@@ -243,43 +243,54 @@ function porosity_correction(
     Kb_S::_T, Ks_S::_T, ρf::_T, ρS::_T, Vs0::_T, depth::_T, α::_T
 ) where {_T<:Number}
     # Empirical porosity-depth model for continental crust after Chen et al., 2020 (hydrogeology journal)
-    m = 0.071;
-    n = 5.989;
-    ϕ0 = 0.474;
+    m = 0.071
+    n = 5.989
+    ϕ0 = 0.474
 
-    ϕ = ϕ0 / ((1.0+depth*m)^n)
+    ϕ = ϕ0 / fastpow((1.0+depth*m), n)
 
     # Takei 1998: Approximation Formulae for Bulk and Shear Moduli of Isotropic Solid Skeleton
-    ν = 0.25                         # poisson ratio
-
-    aij = [
-        0.318 6.780 57.560 0.182
-        0.164 4.290 26.658 0.464
-        1.549 4.814 8.777 -0.290
-    ] 
+    ν = 0.25 # poisson ratio
 
-    bij = [
-        -0.3238 0.2341
-        -0.1819 0.5103
-    ]
+    # Tuples are a better option than standard arrays for matrices/vectors of known size at compile time
+    aij = (
```

Comment:
The same  thing should be done in the function `melt_correction()`, which is currently using arrays.